backend/src/test/resources/rbac-examples/rbac_example_events_notifications_read_access_only.json

```
@@ -0,0 +1,23 @@
+{
+  "meta": {
+    "count": 10,
+    "limit": 25,
+    "offset": 0
+  },
+  "links": {
+    "first": "/api/rbac/v1/access/?application=%2A&limit=10&offset=0",
+    "next": null,
+    "previous": null,
+    "last": "/api/rbac/v1/access/?application=%2A&limit=10&offset=0"
+  },
+  "data": [
+    {
+      "permission": "notifications:*:read:",
```

There's an harmless typo (because we split on `:`) here:
```suggestion
      "permission": "notifications:*:read",
```
Otherwise, LGTM.

---

I changed it myself instead of the commit suggestion. 

Thanks for the hint :)